packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -216,12 +217,12 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
   public componentDidMount(): void {
     if (this._comboBoxWrapper.current) {
       // hook up resolving the options if needed on focus
-      this._events.on(this._comboBoxWrapper.current, 'focus', this._onResolveOptions, true);
+      <EventListener target={this._comboBoxWrapper.current} type="focus" listener={this._onResolveOptions} />;
       if ('onpointerdown' in this._comboBoxWrapper.current) {
         // For ComboBoxes, touching anywhere in the combo box should drop the dropdown, including the input element.
         // This gives more hit target space for touch environments. We're setting the onpointerdown here, because React
         // does not support Pointer events yet.
-        this._events.on(this._comboBoxWrapper.current, 'pointerdown', this._onPointerDown, true);
+        <EventListener target={this._comboBoxWrapper.current} type="pointerdown" listener={this._onPointerDown} />;
       }
     }
```

Comment:
delete this. The "onpointerdown" check seems erroneous.

In places where the condition needs to be considered, you can always conditionalize the target. For example, if there's a "dragging" state that conditionalizes whether we listen to mousemove, you could do this:

```
<EventListener targetRef={ dragging ? rootRef : undefined } ... />
```